cmd/destroy/all.go

```diff
@@ -22,51 +22,35 @@ import (
 	"sync"
 	"time"
 
-	"github.com/okteto/okteto/cmd/utils/executor"
 	"github.com/okteto/okteto/pkg/constants"
 	oktetoErrors "github.com/okteto/okteto/pkg/errors"
-	"github.com/okteto/okteto/pkg/k8s/secrets"
 	oktetoLog "github.com/okteto/okteto/pkg/log"
 	"github.com/okteto/okteto/pkg/model"
 	"github.com/okteto/okteto/pkg/okteto"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
 type localDestroyAllCommand struct {
-	ConfigMapHandler  configMapHandler
```

Comment:
Most of these dependencies were not being used by the `localDestroyAllCommand`. Also, this struct was a dependency of `localDestroyCommand` and these dependencies were being used from there. So I removed the dependencies from here and removed the dependency between `localDestroyAllCommand` and `localDestroyCommand`